frontend/src/features/login/Login.tsx

```diff
@@ -17,6 +17,7 @@ const Login = () => {
   const { showErrorAlert } = useNotifications();
   const login: LoginState = useAppSelector(selectLogin);
   const dispatch = useAppDispatch();
+  const [formIsDirty, setFormIsDirty] = useState(false);
```

Comment:
Curious about the name choice. It doesn't tell much. Why not `formHasBeenSubmitted`?

Reply:
A form is considered "dirty" when a user has modified any of the inputs, as opposed to a "clean" form. It's a common terminology in UI libraries. But in this case I agree it's unclear. I will follow your suggestion.